doc/man7/EVP_MD-MDC2.pod

```diff
@@ -0,0 +1,57 @@
+=pod
+
+=head1 NAME
+
+EVP_MD-MDC2 - The MDC2 EVP_MD implementation
+
+=head1 DESCRIPTION
+
+Support for computing MDC2 digests through the B<EVP_MD> API.
+
+=head2 Identity
+
+This implementation is identified with this name and properties, to be
+used with EVP_MD_fetch():
+
+=over 4
+
+=item "MDC2", "provider=default"
+
+=back
+
+=head2 Gettable Parameters
+
+This implementation supports the common gettable parameters described
+in L<EVP_MD-common(7)>.
+
+=head2 Settable Context Parameters
+
+This implementation supports the following L<OSSL_PARAM(3)> entries,
+settable for an B<EVP_MD_CTX> with L<EVP_MD_CTX_set_params(3)>:
+
+=over 4
+
+=item "pad_type" (B<OSSL_DIGEST_PARAM_PAD_TYPE>) <unsigned integer>
+
+Sets the pad type to be used.
+The only built-in digest that uses this is MDC2.
```

Comment:
Probably this is irrelevant, given this is the MDC2 page.

Reply:
Hahaha, yeah. Will fix